docs/UserGuide.md

```diff
@@ -116,29 +140,36 @@ Example:
 Marks a FYP as "Done"/"In Progress"/"Yet to Start", etc.
 
 Format: `mark i/STUDENT_ID s/STATUS`
+
 * Current supported statuses are as follows:
-  * **"DONE"** - Done
-  * **"IP"** - In Progress
-  * **"YTS"** - Yet To Start
+    * **"DONE"** - Done
```

Comment:
As mentioned in PR, this part of UG already shows the ENums. Hence to tackle issue  #191, should we make it more prominent in help function?

Reply:
Not too sure if we are allowed to do this since its prob a feature flaw if the help message isnt detailed enough?